client/webserver/site/src/html/forms.tmpl

```diff
@@ -1,22 +1,27 @@
 {{define "newWalletForm"}}
 <div class="bg2 px-2 py-1 text-center fs18">
-  Create a
-  <img id="newWalletLogo" class="micro-icon mx-1">
-  <span id="newWalletName"></span>
+  Add a
+  <img id="nwAssetLogo" class="micro-icon mx-1">
+  <span id="nwAssetName"></span>
   Wallet
 </div>
 <div class="p-4">
-  {{if .}}
-    <div class="fs16">{{.}}</div>
+  {{if .subtext}}
+    <div class="fs16">{{.subtext}}</div>
```

Comment:
Can't you treat this text like all of the others and just pass a true/false to the template?

Reply:
Not sure I follow... I want to pass 2 pieces of data to this template, so I'm using a map with optional `subtext` and `showHelp` keys. Couldn't think of a cleaner way to do this, will love any pointers.

Reply:
We're only using the `newWalletForm` template, three places. [Here](https://github.com/itswisdomagain/dcrdex/blob/24a0b3a5fe24b09f296253428ef63802d8f1d1dc/client/webserver/site/src/html/wallets.tmpl#L101), [here](https://github.com/itswisdomagain/dcrdex/blob/24a0b3a5fe24b09f296253428ef63802d8f1d1dc/client/webserver/site/src/html/register.tmpl#L30), and [here](https://github.com/itswisdomagain/dcrdex/blob/24a0b3a5fe24b09f296253428ef63802d8f1d1dc/client/webserver/site/src/html/markets.tmpl#L241). Unless I'm missing something, you're never passing any `subtext` with `showHelp = false`, you're only passing the template argument in one of those places, and `subtext` has no variables in the string. So the `subtext` could be hard-coded into the html template, and it's display also controlled by `showHelp`, just like the other help texts, and you don't need a map, you just need a boolean.

Reply:
Looks like my brain needs a reboot. Seen. Thanks!